.github/workflows/nightly-1.0-builds.yml

```diff
@@ -0,0 +1,142 @@
+name: Nightly Builds (1.0)
+
+on:
+  schedule:
+    - cron: "0 0 * * *"
+  workflow_dispatch:
+
+permissions: {}
```

Comment:
I'm curious what this does?

Reply:
it's an attempt to limit the permissions for the GitHub token - this setting basically says we don't need it at all

Reply:
right, so the default permissions are wider than `{}` (https://docs.github.com/en/actions/using-jobs/assigning-permissions-to-jobs) :+1: great!